md/datetimes-management-tutorial.md

```diff
@@ -0,0 +1,163 @@
+* Create an object with an attribute of type DATE-TIME (TZ)
+* Create a new process
+* Then, in the process data tab, create a new business variable of the type of your business object
+
+### Define the contract and task operations or business variable initialization script
+
+* In the process/task execution tab > contract section, define the contract inputs using the Add from data... button
+* Select the business variable you created previously
+* Follow the steps to generate a contract with a COMPLEX input containing a DATE-TIME (TZ) attribute (keep the option to generate the variable initialization script or the operations (for a task) automatically)
+
+### Generate the form
+
+* Generate the UI designer form by clicking on edit in the form section of the process/task execution tab
+* Since you have a DATE-TIME (TZ) attribute in the contract inputs, a date time picker widget is automatically added to the form. It is configured to handle the user time zone  
+There is nothing else to do.
+
+::: info
+The date time picker widget output is an ISO 8601 formatted String representing the select date and time in UTC (e.g. "2017-01-13T13:00:00Z"). At contract level when received by the process instantiation / task execution REST API, it is converted to a `java.time.OffsetDateTime`.  
+Unless the user is in the GMT time zone, there will be an offset between the time he selects and the 
+:::
+
+### Retrieve the date from the database and display it
+
+In another step form (or in any other page) you may want to retrieve the date and display it.  
+In order to do that:  
+* Create a new task after the existing one
+* Generate its form by clicking on edit in the form section of the task execution tab
+* In the UI Designer, you need first to define a new variable of type external API to retrieve the object containing the date attribute using the context:
+
+![REST API request](images/dates-management-tutorial/tuto-dates-api-request.png) <!--{.img-responsive .img-thumbnail}-->
+
+::: info
+The value returned by the REST API will be an ISO 8601 formatted String.  
+:::
+
+* Drop a date time picker widget in the page and make sure it is configured to handle time zone
+* Set the date time picker value with the variable you just created  
+You may prefer to display it in some text. In order to do that:  
+* Drop a text widget in the page
+* Replace the text widget value with an expression using the uiDate filter to format the date the way you want. E.g., `{{meeting.startDateTime | uiDate:'MM/dd/yyyy h:mm a'}}` (see the embedded documentation for filter for more information on the different formats)
+
+You can now run your process and see that the date retrieved from the business data database is consistent with the date you chose in the previous step.  
+If you change your system time zone while viewing the second step and refresh the page, you will see that the displayed time adapts to the new time zone.
+
+
```

Comment:
Add the fact that the portal doesn't use those types yet but that this is work in progress.